internal/pkg/state/view.go

```diff
@@ -237,14 +253,26 @@ func (v *View) loadPowerActor(ctx context.Context) (*power.State, error) {
 	return &state, err
 }
 
+func (v *View) loadAccountActor(ctx context.Context, a addr.Address) (*account.State, error) {
+	actr, err := v.loadActor(ctx, a)
+	if err != nil {
+		return nil, err
+	}
+	var state account.State
+	err = v.ipldStore.Get(ctx, actr.Head.Cid, &state)
+	return &state, err
+}
+
 func (v *View) loadActor(ctx context.Context, address addr.Address) (*actor.Actor, error) {
-	tree := v.asMap(ctx, v.root)
-	var actr actor.Actor
-	found, err := tree.Get(adt.AddrKey(address), &actr)
+	st, err := treestate.LoadState(ctx, v.ipldStore, v.root)
```

Comment:
Sigh, if you must. Was this necessary?

Reply:
This was something I chanced upon while trying to debug an earlier interop spike (that I then shared with @acruikshank ).  I was using this pattern to inspect actor state which was not loading into the adt and then found that it worked to replace the functionality of the method.  I did not investigate why this was so it may not be necessary, it was only expedient.  However there was a problem with treating the lotus block state as an adt map.

Reply:
Yeah, I'm reverting the change. Seems to be fine.